src/main/java/reposense/commits/model/CommitResult.java

```diff
@@ -63,6 +66,10 @@ public String getHash() {
         return hash;
     }
 
+    public Boolean getIsMergeCommit() {
+        return isMergeCommit;
+    }
+
```

Comment:
The `codecov/patch` test is failing because of a new getter method that is unused. This might require changing the target number in `codecov/patch` CI or I could get rid of the method.

Reply:
yeah sure. You can remove this method